WWTMVC5/WWTMVC5.csproj

```diff
@@ -458,6 +452,18 @@
       <DependentUpon>ResourceService.svc</DependentUpon>
     </Compile>
     <Compile Include="WebServices\ServiceBase.cs" />
+    <Content Include="WWT2\ClientVersion.txt" />
+    <Content Include="WWT2\Copy of updateurl.txt" />
```

Comment:
Can definitely remove `Copy of updateurl.txt` ... just some lax filesystem hygiene there.

Reply:
Probably some old-school version control going on :)